core/src/main/java/com/yahoo/oak/HashChunk.java

```diff
@@ -319,6 +319,64 @@ void printSummaryDebug() {
 
     /********************************************************************************************/
     /*--------------------------------- Iterators Constructors ---------------------------------*/
-    // TODO: update hash iterator later
+    /********************************************************************************************/
+    /*--------------------------------- Iterators Constructors ---------------------------------*/
+
+    /**
+
+     */
+    HashChunk.ChunkIter hashChunkIter(ThreadContext ctx) {
+        return new HashChunk.ChunkIter(ctx);
+    }
```

Comment:
Intellij: Method 'hashChunkIter(com.yahoo.oak.ThreadContext)' is never used 
Intellij: Raw use of parameterized class 'HashChunk' 
Please check Intellij errors/warnings before submitting a PR. If you are not using Intellij, I suggest using it.